### PR TITLE
Fix inline_mode was returning false when explicitly set to true

### PR DESCRIPTION
**Pull request by thii**

### CODE REVIEW

**Comment by thii** [comment]
CI build is failing on unrelated errors.

**Comment by thii** [comment]
@giginet Any chance to take a look into this? Thanks in advance!

**Comment by giginet** [comment]
@thii Thank you for your contribution.

I'm just about to fix CI. Please moment 🙏

File: lib/xcprofiler/plugin.rb [hunk 1]
```diff
@@ -57,8 +57,8 @@ def thresholds
     end
 
     def inline_mode
-      return false if @inline_mode.nil? == false
```
Comment (giginet) [inline lib/xcprofiler/plugin.rb:60]: It's opposite completely 😱

File: lib/xcprofiler/plugin.rb [hunk 2]
```diff
@@ -57,8 +57,8 @@ def thresholds
...
-      true
+      return true if @inline_mode.nil?
+      @inline_mode
```
Comment (giginet) [inline lib/xcprofiler/plugin.rb]: Returning value should be boolean. Could you cast the type?.

`!!@inline_mode`

**Review by giginet — APPROVED** [review]
LGTM 🚀 

Thank you.

**Comment by giginet** [comment]
@thii OK, I've fixed CI. 

https://github.com/giginet/danger-xcprofiler/commit/c795155f62870b6cc4b95faf5b9a22bd2e543f3b

Could you rebase this?

File: lib/xcprofiler/plugin.rb (hunk 2 above)
Comment (thii) [reply lib/xcprofiler/plugin.rb]:
```
Offenses:
lib/xcprofiler/plugin.rb:61:7: C: Avoid the use of double negation (!!).
      !!@inline_mode
      ^
```
RuboCop is not happy with this. What is your suggestion?

**Comment by giginet** [comment]
I think you can disable DoubleNegation violation. 😅 

Could you also modify `.rubocop.yml` ?

```yaml
Style/DoubleNegation 
    Enabled: false
```

**Comment by thii** [comment]
Done! Please release whenever you have a minute.

**Comment by giginet** [comment]
Great work!!! 💯

**Comment by giginet** [comment]
v0.3.1 is released 🎉 
https://rubygems.org/gems/danger-xcprofiler/versions/0.1.0

**Pull request merged**